### ISSUE TITLE
Trial and Error Include fails instantly for every build attempt for Unreal Engine

### ISSUE THREAD

**Issue by jeevcat** [Wumpf/IncludeToolbox#72]
I'm try to use Trial and Error Include on a file in my Unreal Engine project, but it fails instantly for every build attempt with the following error:

```
1>------ Build started: Project: xxx, Configuration: DebugGame_Editor x64 ------
1>The list of distinct item types for all input files is empty.
1>Parameter name: files
========== Build: 0 succeeded, 1 failed, 0 up-to-date, 0 skipped ==========
```

I assume it has something to do with the custom build tool that UE4 uses. Is there any way for me to debug the build command that IncludeToolbox is using?

**Comment by Wumpf** [comment]
Yes, it's looking at MSVC's compiler settings and methods whereas Unreal uses a custom build as far as I know. Not sure how hard it would be to get it to work

**Comment by jeevcat** [comment]
Would it be possible to have an option to run an arbitrary custom command instead of the usual attempt to compile?

**Comment by jeevcat** [comment]
Or even to just attempt to build the entire project? Very slow, but there for compatibility?
